src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -343,7 +357,7 @@ public Visitable visit(Parameter parameter, Void p) {
     try {
       parameter.resolve().describeType();
       return super.visit(parameter, p);
-    } catch (UnsolvedSymbolException e) {
+    } catch (Exception e) {
```

Comment:
Why the change to catch an arbitrary exception? I'd prefer to avoid catching `Exception`, since that will also catch e.g., `NullPointerException` and other run time problems, not just unsolved symbols. If there are two kinds of exceptions you want to catch, you can use the `|` syntax described here: https://docs.oracle.com/javase/8/docs/technotes/guides/language/catch-multiple.html